README.md

```diff
@@ -92,7 +104,17 @@ The Admin sign in through sign-in page and he will be redirected to the dashboar
 
 ## The MVP :rainbow:
 
- GIF Image
+ - Web
+ 
+![](https://i.imgur.com/KtYlkTi.gif)
+
+![](https://i.imgur.com/CaNT7yA.gif)
```

Comment:
good picture, but I have a silly suggestion to try to crop the black line at the top.

Reply:
good Notice!